pandas/core/reshape/pivot.py

```diff
@@ -598,3 +623,68 @@ def _get_names(arrs, names, prefix='row'):
             names = list(names)
 
     return names
+
+
+def _pivot_simple(index, columns, values):
+    """
+    Produce 'pivot' table based on 3 columns of this DataFrame.
+    Uses unique values from index / columns and fills with values.
+    Parameters
+    ----------
+    index : ndarray
+        Labels to use to make new frame's index
+    columns : ndarray
+        Labels to use to make new frame's columns
+    values : ndarray
+        Values to use for populating new frame's values
+    Notes
+    -----
+    Obviously, all 3 of the input arguments must have the same length
+    This is ONLY used for testing in pandas/test/test_panel.py
+    Returns
+    -------
+    DataFrame
+    See also
+    --------
+    DataFrame.pivot_table : generalization of pivot that can handle
+        duplicate values for one index/column pair
+    """
+    if (len(index) != len(columns)) or (len(columns) != len(values)):
+        raise AssertionError('Length of index, columns, and values must be the'
+                             ' same')
+    if len(index) == 0:
+        return DataFrame(index=[])
+    hindex = MultiIndex.from_arrays([index, columns])
+    series = Series(values.ravel(), index=hindex)
+    series = series.sort_index(level=0)
+    return series.unstack()
+
+
+def _slow_pivot(index, columns, values):
```

Comment:
I don't think we are using this *anywhere* but the ``test_panel`` test? if so I would just blow it away & remove the test